src/Engine/RadiumEngine.hpp

```diff
@@ -289,6 +289,8 @@ class RA_ENGINE_API RadiumEngine
     uint getFrame() const;
     /// \}
 
+    std::string getRadiumResourcesDir() { return m_resourcesRootDir; }
```

Comment:
Two things : 
 - The Engine is a singleton and the Radium resources are not linked to the engine but to the whole Radium ecosystem. So, this method might be static, with m_resourcesRootDir being also static and initialized/checked when the singleton is created.
 - As this correspond to _internal_ radium resources, perhaps call the method `getRadiumInternalResourcesDir()` or something more clear about the _Radium Bundled resources_ semantic.

Reply:
Static data member initialized by "dynamic" singletons ?
I do not get the point here.
And two notes:
I think of using `RadiumEngine->getInstance()->getResourcesDir()` is a simple and meaningful alternative (the resources of RadiumEngine)
I do not get the idea of "ecosystem" used several time in the code/doc.
This function is more like a cache on the Engine side.

Reply:
OK, I'll replace all the "ecosystem" into the code/doc in a next PR.